test/smoke_test/bcda_client.go

```diff
@@ -99,6 +99,7 @@ func startJob(endpoint, resourceType string) *http.Response {
 
 	req.Header.Add("Prefer", "respond-async")
 	req.Header.Add("Accept", "application/fhir+json")
+	req.Header.Add("Accept-Encoding", "gzip")
```

Comment:
Is this the right spot to add this?  I thought we would need to add it to the `get` method down around line 112, so that the header is set when we download the files.